react_main/src/pages/Game/Game.jsx

```diff
@@ -1868,12 +1868,19 @@ export function PlayerList(props) {
     const history = props.history;
     const stateViewingInfo = history.states[props.stateViewing];
     const alivePlayers = Object.values(props.players).filter(p => {
-        if (props.stateViewing > -1 && stateViewingInfo.deaths[p.id]) {
-            return !stateViewingInfo.deaths[p.id].dead && !p.left;
+        let isDead = false;
+        if (stateViewingInfo.deaths[p.id]) {
+            isDead = stateViewingInfo.deaths[p.id].dead;
         }
-        return true;
+        return !isDead && !p.left;

```

Comment:
I personally feel that adding support for reviewing old games is overkill, unless it happens frequently

Reply:
This change was for new games, because deaths aren't initialized on Pregame

So for Pregame player list, it needs to check if deaths for playerIds exists anyway

All old games are also compatible with new history system, they just have a nulled time of death value